README.md

```diff
@@ -1,7 +1,22 @@
-# Carapace
+# ![Carapace](./screenshot.png)
```

Comment:
The logo seems a big big. maybe sub is a smaller version? 
![image](https://user-images.githubusercontent.com/2371345/65792037-4f388380-e139-11e9-83ae-501282063c6e.png)

Reply:
this is the logo on the repo...I can try to shrink it and drop in a smaller version...I have Photoshop :)

Reply:
thanks for the link...I didn't realize that was the link until I actually thought about it so I used yours